addons/website_sale/static/src/xml/website_sale_dashboard.xml

```diff
@@ -29,8 +29,14 @@
                 <div class="col-12 row o_box">
                     <t t-if="widget.dashboards_data.sales.summary.order_count">
                         <h2 class="col-lg-7 col-12">
-                            <t t-if="widget.date_range">
-                                Sales Since Last <t t-esc="widget.date_range"></t>
+                            <t t-if="widget.date_range=='week'">
+                                <h2>Sales Since Last Week</h2>
```

Comment:
or not :/ Why add a h2 here...

h2 into h2 break the layout :'(

Will be fixed by https://github.com/odoo/odoo/pull/60737